src/frameworks/foundation.rs

```diff
@@ -55,7 +55,7 @@ pub struct State {
 pub type NSInteger = i32;
 pub type NSUInteger = u32;
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Was this an accidental change or something?

Reply:
Ah, it's not accidental, but it could be removed if my suggestion below can be implemented: https://github.com/hikari-no-yume/touchHLE/pull/154/files#r1321731604.